spyder/plugins/breakpoints/plugin.py

```diff
@@ -46,17 +46,19 @@ def __init__(self, parent=None):
         self.setLayout(layout)
 
         self.breakpoints.set_data()
-    
+
+        path = osp.join(self.PLUGIN_PATH, self.IMG_PATH)
+        self.icon = ima.icon('breakpoints', icon_path=path)
+
     #------ SpyderPluginWidget API --------------------------------------------
     def get_plugin_title(self):
         """Return widget title"""
         return _("Breakpoints")
-    
+
     def get_plugin_icon(self):
         """Return widget icon"""
-        path = osp.join(self.PLUGIN_PATH, self.IMG_PATH)
-        return ima.icon('profiler', icon_path=path)
-    
+        return self.icon

```

Comment:
@ccordoba12 Probably should remove the `get_plugun_icon` method altogether then? Will remove quite a bit of icons from `icon_manager` as well.

Reply:
What Carlos meant to say is that qe do not use icons for toggling plugin panes? The panes menu has no icons.  This one is located on a different menu for convenience so it is being duplicated? 

We DO use icons for plugins in the preferences, so there is. O plan to remove any method.

Reply:
That haven't changed here. I only added an icon for the debug menu (it does not show up for the toggling anyway). And as shown in my recent comment, two out of four plugins accessible from the menus do have icons already.

I was trying to make it more consistent (and correct, by not using the profiler icon...). But if there are other opinions, I'm not the one to decide.